docs/pe/user-guide/ui/users.md

```diff
@@ -4,8 +4,155 @@ assignees:
-{% include docs/user-guide/ui/users.md %}
\ No newline at end of file
+* TOC
+{:toc}
+
+## Introduction
+
+User is an entity that can log in to the ThingsBoard web interface, execute REST API calls, access devices, assets and other entities if they have permissions to do so.
+Users are grouped into user groups.  
+By default, there are two autogenerated user groups: 
+[the Tenant Administrators](/docs/pe/user-guide/ui/users/#tenant-administrator-user-group) and [the Tenant Users](/docs/pe/user-guide/ui/users/#tenant-user-user-group).
+Tenant administrator can directly create new users in autogenerated user groups, however please note that they will have the access to all dashboards as tenant administrator that has created them.
+Also, there is the "All" user group. If a user is created directly in the "All" group, he won't have any permissions and will view nothing.
+
+{% include images-gallery.html imageCollection="users-intro-pe" showListImageTitles='true' %}
+
+**Note**: outgoing email settings have to be properly configured using [system administrator](/docs/pe/user-guide/ui/tenants/) account. This is required to send activation email to the new users.
+
+## User groups
+
+A User group is group of users of the same level with the same permissions. One user can simultaneously belong to several user groups.
+
+##### Tenant Administrator user group
+
+The created tenant administrators in the Tenant Administrator user group have the same permissions as the tenant administrator who added them to the administrators group (except they can not delete their administrator).
+Let's create a tenant administrator to see what he can do.
+
+{% include images-gallery.html imageCollection="user-bob-pe" showListImageTitles='true' %}
+
+##### Tenant User user group
+
+Tenant users group has read-only permissions. Let's create a tenant user to see what he can do.
+
+{% include images-gallery.html imageCollection="user-alice-pe" showListImageTitles='true' %}
+
+##### Adding a new user group
+
+For the tenant administrator it is possible to create your own user group with customized permissions. 
+Create a user group and add the previously created [Roles](/docs/pe/user-guide/rbac/#roles) in the Roles tab in the user group details to configure users permissions in the new group.
+
+{% include images-gallery.html imageCollection="user-groups-pe" showListImageTitles='true' %}
+
+## Edit user group
+
+Like any group of entities, the user group can be easily customized and edited.
+To edit a user group, click the user group row to enter details about the user group.
+
+{% include images-gallery.html imageCollection="user-edit-pe" showListImageTitles='true' %}
+
+Learn more about [cell style function](/docs/pe/user-guide/ui/advanced-data-key-configuration/#12-cell-style-function).
+
+Learn more about [actions](/docs/pe/user-guide/ui/widget-actions/).
+
+## Delete user group
+
+User group can be deleted by the administrator who created it.
```

Comment:
A user group may be deleted by the user with sufficient permissions. Please note that the users that belong to a group will not be deleted. A single user may be a member of multiple user groups and is always a member of the special group "All".

Reply:
corrected.

p.s. "may" is not a substitute for "can". It is incorrect to use "may" as a physical ability.